worlds/tunic/er_rules.py

```diff
@@ -403,6 +409,13 @@ def set_er_region_rules(world: "TunicWorld", regions: Dict[str, Region], portal_
         connecting_region=regions["Forest Belltower Lower"],
         rule=lambda state: has_ladder("Ladder to East Forest", state, world))
 
+    regions["Forest Belltower Main behind bushes"].connect(
+        connecting_region=regions["Forest Belltower Main"],
+        rule=lambda state: can_get_past_bushes(state, world))
```

Comment:
This is also possible with Magic Orb + Magic Dagger by waiting for a slime to come over to you, using the dagger to freeze it, and then grappling it with the orb. Should probably require easy ice grapple logic.